projects/clas12MC.yaml

```diff
@@ -0,0 +1,11 @@
+Name: clas12MC
```

Comment:
```suggestion
Name: JLab_clas12MC
```

Perhaps? @lmichael107

Reply:
This is for an existing project which was not registered